shim/src/main.rs

```diff
@@ -446,6 +448,24 @@ fn get_version() -> &'static str {
 }
 
 fn main() -> Result<()> {
+    // custom footer for update message
+    let footer = format!(
+        "Follow {username} for updates: {url}",
+        username = "@turborepo".gradient([RGB::new(0, 153, 247), RGB::new(241, 23, 18)]),
+        url = "https://twitter.com/turborepo"
+    );
+
+    // check for updates
+    check_for_updates(
+        "turbo",
+        "https://github.com/vercel/turbo",
+        Some(&footer),
+        get_version(),
+        None,
+        None,
+    )
+    .ok();
```

Comment:
```suggestion
    let _ = check_for_updates(
        "turbo",
        "https://github.com/vercel/turbo",
        Some(&footer),
        get_version(),
        None,
        None,
    );
```

I feel like that's a bit more explicit about ignoring the result